### PR TITLE
feat: #27 add group actions to proposal create flow

### PR DESCRIPTION
**Pull request by blushi**
closes: #27 
closes: #48

### CODE REVIEW

**Comment by blushi** [comment]
@erika you can test this on https://deploy-preview-98--regen-groups-ui.netlify.app/

**Comment by blushi** [comment]
> I wonder if we should add a tooltip here to explain the create proposal button? https://monosnap.com/file/NGx0eDkewXD9mF3HchJdBZ4XCf5VRM Or maybe change the button to "create proposal to save"? Or maybe both, wdyt?

Yeah I can add that quickly

> Also, this could be a separate task but we should probably warn the user if they navigate away without clicking "create proposal"

Yes probably worth a separate task since 1. this PR is already quite large and 2. we might want the same behavior too even when the policy is not the admin

**Comment by erikalogie** [comment]
> > I wonder if we should add a tooltip here to explain the create proposal button? https://monosnap.com/file/NGx0eDkewXD9mF3HchJdBZ4XCf5VRM Or maybe change the button to "create proposal to save"? Or maybe both, wdyt?
> 
> Yeah I can add the tooltip quickly, but afraid that if we change the wording, it might result in a too long button visually and also by creating a proposal, it won't actually be saved, until the proposal passes and is executed, so it could be a bit misleading IMO

Yes that's true, let's just do the tooltip for now
> 
> > Also, this could be a separate task but we should probably warn the user if they navigate away without clicking "create proposal"
> 
> Yes probably worth a separate task since 1. this PR is already quite large and 2. we might want the same behavior too even when the policy is not the admin

Ok cool I'll add a task for this.

File: src/pages/proposal-create-page.tsx [hunk 1]
```diff
@@ -72,33 +118,38 @@ export default function ProposalCreate() {
...
         metadata: { title, summary },
         proposers: [account.address],
+        groupId: group.id.toString(),
       })
       if (!data?.proposalId)
         throwError('Proposal transaction completed, but no proposal ID found')
       toastSuccess(data.transactionHash)
       return data.proposalId
     } catch (error) {
+      console.log(error)
```
[the remark this answers is not in the document]
Comment (blushi) [reply src/pages/proposal-create-page.tsx]: good catch, removed: https://github.com/regen-network/groups-ui/pull/98/commits/dd3684285207fc908fa2fef91597952587ba65e6

**Comment by ryanchristo** [comment]
@erikalogie can you give the final ok on this? Does the tooltip look good to you?

**Comment by erikalogie** [comment]
This looks good!

**Pull request merged**